packages/services/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
Why not to have a single file on root?

Reply:
yeah, this would be the best, also this would affect other packages too.